### PR TITLE
Implemented rooms, refactored AV selection

### PR DESCRIPTION
**Pull request by kmilan-lwdr**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
 const express = require("express");
 const app = express();
 
+const TIMEOUT = 600000; // 10 minutes 600000
```
Comment (anvuori) [inline server.js:4]: 👍

File: server.js [hunk 2]
```diff
@@ -26,36 +30,103 @@ io.sockets.on("error", (e) => console.log(e));
...
+      let room = new Map();
+      room.set(BROADCASTER, socket.id);
+      rooms.set(tokenId, room);
+    }
+    socket.roomId = tokenId;
+    socket.join(tokenId);
+    console.log(rooms);
+    socket.to(tokenId).emit(BROADCASTER);
+    socket.isBroadcaster = true;
+    //broadcaster = socket.id;
```
Comment (anvuori) [inline server.js:53]: do we need the commented code?

**Review by anvuori — APPROVED** [review]
Very nice 👍

File: server.js (hunk 2 above)
Comment (kmilan-lwdr) [reply server.js:53]: Nah actually not

**Pull request merged**